common/transform/src/main/java/ai/h2o/mojos/deploy/common/transform/MojoScorer.java

```diff
@@ -92,6 +99,16 @@ public ScoreResponse score(ScoreRequest request) {
     MojoFrame responseFrame = doScore(requestFrame);
     ScoreResponse response = scoreResponseConverter.apply(responseFrame, request);
     response.id(pipeline.getUuid());
+
+    if (request.getRequestShapleyValueType() == null) {
+      return response;
+    }
```

Comment:
If someone has Shap values disabled, and explicitly specifies `NONE` for Shap values, I'd imagine some would argue that the request should succeed.

The additions in this branch would fail those requests.

Reply:
Fixed this, thanks @orendain